### ISSUE TITLE
cmd/gofmt: not supported by windows

### ISSUE THREAD

**Issue by alexbrainman** [golang/go#18026]
### What version of Go are you using (`go version`)?
go version devel +50c4dbc Wed Nov 23 00:44:01 2016 +0000 windows/386

### What operating system and processor architecture are you using (`go env`)?
set GOARCH=386
set GOBIN=
set GOEXE=.exe
set GOHOSTARCH=386
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=c:\dev
set GORACE=
set GOROOT=c:\dev\go
set GOTOOLDIR=c:\dev\go\pkg\tool\windows_386
set GCCGO=gccgo
set GO386=
set CC=gcc
set GOGCCFLAGS=-m32 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\DOCUME~1\brainman\LOCALS~1\Temp\go-build246447175=/tmp/go-build -gno-record-gcc-switches
set CXX=g++
set CGO_ENABLED=1
set PKG_CONFIG=pkg-config
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2

### What did you do?
```
C:\dev\src\a>dir /b
foo.go

C:\dev\src\a>type foo.go
package main

import (
    "log"
    "os"
)

func main() {
    _, err := os.Stdin.Stat()
    if err != nil {
        log.Fatalln(err)
    }
}
C:\dev\src\a>go fmt
foo.go
chmod foo.go.662713035: not supported by windows
exit status 2

C:\dev\src\a>
```

### What did you expect to see?
I expect to see no output from gofmt

### What did you see instead?
gofmt failed with error message. Chmod is not implemented on Windows, so it is failing.

Alex

**Comment by bradfitz** [comment]
@griesemer, this is a regression from your #8984 fix (b188b4cc110261a004674df5a4e209cc4894d314).

**Comment by bradfitz** [comment]
Actually, just sent https://go-review.googlesource.com/33477
